test/stability/testSeparate/testSeparate.go

```diff
@@ -55,6 +55,7 @@ var (
 	recvCount2 uint64
 
 	count         uint64
+	countRes      uint64
```

Comment:
We should change count anytime, because we generate packet's difference from it. And countRes is count for time more than T.

(if you don't change count you will have a lot of packets  in the very beginning of T at receive side which will be the same. They will be calculated as if separate doesn't work)

Reply:
We skip packets in the very beginning

Reply:
Moment T is achieved simultaneously for send and for receive. If you send the same packets before T, receive will receive the same packets by "T + latency" time. And it will think that separate doesn't work, because all the packets are the same. It means that you need to change packets even before "T". But calculate them only after "T".